OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            xml:lang="gez">ዘቀዳሚት</q> (<locus target="#66v"/>), <q
+                                            xml:lang="gez">ዘዘወትር</q> (<locus target="#72r"/>, in
+                                        black), <q xml:lang="gez">ዘእኁድ</q> (<locus target="#74r"/>)
+                                    </desc>
+                                </item>
+
+                                <item xml:id="e2">
+                                    <desc> Scribal corrections occur occasionally over erasure (e.g.
+                                            <locus target="#112r"/> here comprising several lines)
+                                        or interlineally (e.g. <locus target="#28v #31v"/>), also
+                                        made in a secondary hand (e.g. <locus
+                                            target="#29v #40v #88v #114r"/>). Spaces (after erasure)
+                                        were filled with an interlaced band on <locus
+                                            target="#50r #72r"/> and with several thin lines on
+                                            <locus target="#66v #79v"/>. The scribe wrote the last
+                                        two lines of <ref target="#ms_i1.3"/> in small script in
+                                        order to complete the section on the same folio <locus
+                                            target="#35v"/>. Writing exercises were executed on
+                                            <locus target="#123r"/> and upside down on <locus
+                                            target="#121v #122v"/>. The name <persName
+                                            ref="PRS13017GabraMar"/>is written on <locus
+                                            target="#3r"/> (the rest of the entire folium is
+                                        blank).</desc>
+                                </item>
+
+                                <item xml:id="e3">
+                                    <locus target="#2r"/>
+                                    <desc>A turquoise and gold label of the textile trading company
+                                        A.H. Huschke &amp; Co with a leaping horse is sewn onto
+                                            <locus target="#2v"/>. The India-based company was
+                                        active in the 19th century until its founder, August Henry
+                                        Huschke of Swiss-German family background, died in 1866 and
+                                        his company was sold. A.H. Huschke &amp; Co acquired a high
+                                        ‘reputation for luxury cottons’ and other ‘quality goods for
+                                        the fashion market’</desc>
+                                    <note>(see: <bibl>
+                                            <ptr target="bm:Nenadic2019"/>
```

Comment:
just for the future, the zotero tag should be NameYearKeyword (something from the title of the reference)